src/posix/platform/multicast_routing.hpp

```diff
@@ -74,6 +74,12 @@ class MulticastRoutingManager : public Mainloop::Source, private NonCopyable
      */
     void Init(otInstance *aInstance);
 
+    /**
+     * This method shuts down the Multicast Routing manager.
```

Comment:
```suggestion
     * This method deinitializes the Multicast Routing manager.
```

Reply:
@bukepo The documentation is tedious, I remember that we want to avoid such documents. Maybe just remove the documentation here.

Reply:
Removed.